hack/godep-save.sh

```diff
@@ -21,9 +21,27 @@ set -o pipefail
 FRAKTI_ROOT=$(dirname "${BASH_SOURCE}")/..
 source "${FRAKTI_ROOT}/hack/lib/init.sh"
 GODEP="${GODEP:-godep}"
+PRE_RESTORE_FLAG="--pre-restore"
 
+# Some special pkg dependencies we may need restore them
+# before godep-save when try to update
+PRE_RESTORE_PKG=(
+  "k8s.io/client-go"
+)
 
-cd ${FRAKTI_ROOT}
+if [ $# -gt 0 ]; then
```

Comment:
it related with inner `else` section, could remove them all or combine with it like `elif [ $# -gt 0 ]` if you like.